docs/stencil-docs/javascript-and-event-hooks/adding-javascript.md

```diff
@@ -338,88 +259,45 @@ export default class Cart extends PageManager {
         }
 ```
 
----
-
-<a href='#adding_mapping-custom' aria-hidden='true' class='block-anchor'  id='adding_mapping-custom'><i aria-hidden='true' class='linkify icon'></i></a>
-
-## Mapping Custom Templates to JavaScript Modules
+## Mapping custom templates to JavaScript modules
 
 If you add [custom page templates](/stencil-docs/storefront-customization/custom-templates/) to your theme, you can edit the same `assets/js/app.js` file to map each custom template to an appropriate JavaScript module.
 
 In the following example, we will map a custom JavaScript file, `assets/js/themes/custom.js` to a custom product page file `templates/pages/custom/product/customProd.html`.
 
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name">Custom JS file</div>
-    </div><div class="HubBlock-header-subtitle">assets/js/themes/custom.js</div>
-</div>
-
-<!--
-title: "Custom product page file"
-subtitle: "assets/js/themes/custom.js"
-lineNumbers: true
--->
-
-
 ```js
 import PageManager from "./page-manager";
 
-export default class Custom extends PageManager  {
-
+export default class Custom extends PageManager {
     onReady(){
         alert("Hello world!");
     }
 }
-
 ```
 
-This is a basic module that creates a class called Custom which extends the PageManager class. 
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name">Custom product page template file</div>
-    </div><div class="HubBlock-header-subtitle">templates/pages/custom/product/customProd.html</div>
-</div>
+This is a basic module that creates a class called `Custom` which extends the PageManager class.
 
-<!--
-title: "Custom product page template file"
-subtitle: "templates/pages/custom/product/customProd.html"
-lineNumbers: true
--->
 ```html
 {{~inject 'template' template}}
-    <h2>Hello World!</h2>
-    <body>
-    Some custom content!
-    <body>
-    <script>window.__webpack_public_path__ = "{{cdn 'assets/dist/'}}";</script>
-    <script src="{{cdn 'assets/dist/theme-bundle.main.js'}}"></script>
-    <script>window.stencilBootstrap("{{page_type}}", {{jsContext}}).load();</script>
+<h2>Hello World!</h2>
+<body>
+Some custom content!
+<body>
+<script>window.__webpack_public_path__ = "{{cdn 'assets/dist/'}}";</script>
+<script src="{{cdn 'assets/dist/theme-bundle.main.js'}}"></script>
+<script>window.stencilBootstrap("{{page_type}}", {{jsContext}}).load();</script>
 ```
 
-In order to successully map your custom module to a custom template file, that file must
-* Inject the custom template 
-* Load webpack 
+In order to successully map your custom module to a custom template file, that file must do the following:
```

Comment:
Do the following need to be done in this specific order? If so, this should be an ordered list.

Reply:
Nah, its just a general list of requirements.